includes/class-wsuwp-transfer-equivalencies.php

```diff
@@ -26,5 +36,528 @@ public static function get_instance() {
 	 *
 	 * @since 0.0.1
 	 */
-	public function setup_hooks() {}
+	public function setup_hooks() {
+		add_action( 'init', array( $this, 'register_content_type' ), 10 );
+		add_action( 'add_meta_boxes_' . $this->content_type_slug, array( $this, 'add_meta_boxes' ), 10 );
+		add_action( 'admin_menu', array( $this, 'admin_menu' ) );
+		add_shortcode( 'tce_search', array( $this, 'display_tce_search' ) );
+		add_shortcode( 'tce_interface', array( $this, 'display_tce_interface' ) );
+		add_action( 'wp_enqueue_scripts', array( $this, 'wp_enqueue_scripts' ) );
+		add_action( 'wp_ajax_nopriv_tce_navigation', array( $this, 'ajax_callback' ) );
+		add_action( 'wp_ajax_tce_navigation', array( $this, 'ajax_callback' ) );
+	}
+
+	/**
+	 * Register a content type to track information about transfer credit equivalencies.
+	 */
+	public function register_content_type() {
+		$labels = array(
+			'name' => 'Institutions',
+			'singular_name' => 'Institution',
+			'all_items' => 'Institutions',
+			'view_item' => 'View Institution',
+			'add_new_item' => 'Add New Institution',
+			'edit_item' => 'Edit Institution',
+			'update_item' => 'Update Institution',
+			'search_items' => 'Search Institutions',
+			'not_found' => 'No Institutions found',
+			'not_found_in_trash' => 'No Institutions found in Trash',
+		);
+
+		$args = array(
+			'labels' => $labels,
+			'description' => 'Transfer institutions',
+			'public' => true,
```

Comment:
I *think*, (*think*), we can do:

```
'public' => false,
'publicly_queryable' => true,
'exclude_from_search' => true,
'show_in_rest' => true,
'show_ui' => true,
```

...which should make institutions available through the REST API and the admin interface, but hide them from the front end as `/institution/....` and from search results. This would involve dropping the `rewrite` argument as well.